src/openrct2/object/Object.h

```diff
@@ -364,11 +367,11 @@ class Object
 
     const std::vector<std::string>& GetAuthors() const;
     void SetAuthors(std::vector<std::string>&& authors);
-    const std::string& GetVersion() const
+    const ObjectVersion& GetVersion() const
```

Comment:
Since this now only 6 bytes big, I guess we have to get of the reference here and let it copy, @ZehMatt ?

Reply:
I'd just stick with the const reference as a return type for two reasons. We can't assume the size of `ObjectVersion`. It's possible that the tuple will need to have padding to meet the alignment requirements of the uint16_t type for a target platform. The other reason is that returning a const reference cannot be considered the owner of the data.

Reply:
This isn't performance critical unless the its called each frame which I somewhat doubt so there is not really a right or wrong way. One should keep in mind that OpenRCT2 has still 32 bit builds so that value won't fit a single register there only on 64 bit architectures so const ref should be the safe way to go about it.